ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -254,3 +254,8 @@ platform_speed_groups:
   7280R:             # Only setting speed-groups on 7280R platform, so only L3leaf should get this setting.
     25G: [ 3, 2 ]    # Unsorted order, but we should sort output correctly.
     10G: [ 1, 2, 4 ] # Duplicate speed-group 2. Since we sort on key first the result will be 25G for group 2
+
```

Comment:
Do we really need to set this on all these devices? It's easier to review if we keep the scope smaller, so maybe one or 2 devices is enough.